### PR TITLE
Update M17 theme footer

### PR DESCRIPTION
**Pull request by bridgetburke**
Added flex display and additional CSS to solve the widgets and the footer text stacking. Removed extraneous widget CSS from parent theme.

Ticket: MICHELP-31

### CODE REVIEW

File: wp-content/themes/mlibrary-twentyseventeen/style.css [hunk 1]
```diff
@@ -119,9 +121,11 @@ h1, h2, h3, h4, h5, h6 {
   color: white;
 }
 
-/**
- * Additional styling for the gist embed.
- */
-.gist {
-  margin-bottom: .25em;
```
Comment (bertrama) [inline wp-content/themes/mlibrary-twentyseventeen/style.css]: This looks unrelated to the footer, was it intentional?

Comment (bridgetburke) [reply wp-content/themes/mlibrary-twentyseventeen/style.css]: This was not intentional. I meant to ask you about if we were using the embeds anymore and then was distracted and must have deleted the selection. I can add it back for now.

**Comment by bertrama** [comment]
I don't think it's in use by any sites using this theme right now.  The design lab blog has the gist shortcode in some posts, and I didn't want them to look weird should they decide to change themes.

**Comment by bertrama** [comment]
That said, if you think it would be better to have the plugin that provides the short-code processing to also provide the css, I can make that change.  I thought it would be better if the themes could set their own values for styles.

**Comment by bridgetburke** [comment]
It's back in and I'm good with it being in the theme CSS. I added it to my list of things to address when I look over the theme as a whole.

**Review by bertrama — APPROVED** [review]
